h/services/http.py

```diff
@@ -0,0 +1,65 @@
+from requests import RequestException, Response, Session  # noqa: A005
+
+from h.services.exceptions import ExternalRequestError
+
+HTTP_SERVICE_TIMEOUT = 10
+
+
+class HTTPService:
+    """Send HTTP requests with `requests` and receive the responses."""
+
+    def __init__(self) -> None:
```

Comment:
```suggestion
    def __init__(self):
```

Is the type annotation necessary here, or is it just visual noise? `__init__()` methods must always return `None`

Reply:
Well, it's kind of subtle like they say but it seems to be the recommended approach.
https://peps.python.org/pep-0484/#the-meaning-of-annotations